server/publications.coffee

```diff
@@ -73,7 +73,7 @@ Meteor.publish 'studentLessonData', (username) ->
     fields:
       points: 1
       username: 1
-      status: 1
+      status.online: 1
```

Comment:
Oh, I did not notice this before, status.online should be wrapped in quotes: 'status.online' 

It's fixed here: https://github.com/Elfoslav/codermania/commit/5f08a96f0e33f0052d860bdfd7f57a2c4a16e390